src/transmission_manager.rs

```diff
@@ -281,13 +285,14 @@ mod test {
             );
         }
         // TODO: figure out way to check math for n_attempts > 1
```

Comment:
Are you going to address this TODO.

Generally, I would prefer we use `TODO(<id>)` rather than just `TODO`. E.g., `TODO(cym4@cdc.gov)` or `TODO(cym4)`